pages/api/allergens.js

```diff
@@ -0,0 +1,21 @@
+import client from "../../helpers/db";
+
+export default async function handler(req, res) {
+  if (req.method === "GET") {
+    try {
+      await client.connect();
+      const database = client.db("devdb");
```

Comment:
Try to abstract the database name so that it's not hard-coded on different parts of the code.